### PR TITLE
docs(progress-steps): progress steps component RFC

### PR DESCRIPTION
**Pull request by nadiia**
screen shot 2018-09-20 at 9 10 10 am

### CODE REVIEW

File: rfcs/progress-steps-component.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import {ProgressSteps, StyledNumberedStep} from 'baseui/progress-steps';
+
+export default () => {
+  const overrides ={
+    Step: props => <StyledNumberedStep {...props}/>,
+    StepTitle: props => <CustomTitle {...props} />,
+  };
+  return
+    <ProgressSteps
+      onClick={()=>{console.log('step is clicked')}}
```
Comment (schnerd) [inline rfcs/progress-steps-component.md]: will onclick be called with the step index?

Comment (nadiia) [reply rfcs/progress-steps-component.md]: yes! updated

**Comment by abmai** [comment]
Thanks for knocking this out!!

How do you see this supporting the last case, would additional internal items just be children of `Step`?

image

**Comment by nadiia** [comment]
@abmai I was thinking those would be just children of a `Step`. Do you think we should rather define these type of children through the props or maybe just provide additional styled components like `StyledStepSubtitle`, `StyledStepAction`?

**Comment by abmai** [comment]
> @abmai I was thinking those would be just children of a `Step`. Do you think we should rather define these type of children through the props or maybe just provide additional styled components like `StyledStepSubtitle`, `StyledStepAction`?

I think providing more styled components is nice, maybe not the action since it's just a button but `<StyledStepSubtitle>` and `<StyledStepText>` would be good to have. What do you think?

**Comment by nadiia** [comment]
Added the two mentioned to the exports list ^

**Pull request merged**